packages/fx-core/tests/plugins/solution/solution.scaffold.test.ts

```diff
@@ -234,11 +234,8 @@ describe("Solution scaffold() reading valid manifest file", () => {
         path.join(testFolder, "./templates/azure/provision", "simpleAuthProvision.bicep")
       )
```

Comment:
update frontendHosting and simpleAuth file name as identity

Reply:
later do it